styles.css

```diff
@@ -112,15 +128,26 @@ a {
   height: 25px;
 }
 
-.cards-container {
+/* .cards-container {
   background-color: #e5e5e5;
   padding: 11rem 1.5rem;
   display: grid;
   grid-template-rows: repeat(3, 1fr);
   gap: 12rem;
+} */
+
+.cards-container {
+  background-color: #e5e5e5;
+  padding: 11rem 1.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 12rem;
```

Comment:
- [ ] Kindly follow the design template for the `cards` section. For the template, each card is divided into two, the card picture on one side and the description and other details on the other. I would advise you to use the CSS grid layout to style this section. 👍  

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/213261903-444cac6c-dab7-45db-a1cc-5c1c2fcc6497.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/213262049-d1abf8f3-3471-4dd2-96f0-f752661329c1.png)